### ISSUE TITLE
Minio Helm chart is deprecated in favour of an operator

### ISSUE THREAD

**Issue by tico24** [sorry-cypress/charts#45]
We will have to follow suit at some point.

https://github.com/minio/charts/tree/master/minio

**Comment by iorlas** [comment]
Also, I should mention, the full stack is basically unusable for some audiences. My case goes here:
- Azure AKS, hence K8s
- The VM is limited to 7 gb of RAM, which includes all the applications and sorry cypress stack
- I heed to store the recordings, it is mission-critical. MinIO feels awesome at what it does, but I'm unable to get the advantage of it.

The problems I have:
- When minio is enabled, it starts the chart which requires... 4gb of RAM. I'm unable to dedicate such an amount of RAM to one service.
- I'm unable to pass custom values downstream thru the sorry cypress values file

So, right now, I have only these options:
- Fork the chart, redo it to play nicely with separately installed Minio instance (every time you fork something, you have to maintain it, but I don't have enough resources, no matter how much I wanted to help the project)
- Create AWS account to allocate some S3 storage (too many accounts, separate billing, might be too expensive)
- Create separate k8s node for the minio with at least 4gb dedicated to it (expensive)

Yeah, I won't be using Sorry Cypress for now, but I hope some details will help you push the product further!

**Comment by tim-sendible** [comment]
You absolutely don't have to fork the chart to change the ram. If it's an option in the minio chart, you just add it to your values.yaml. 

If it's not an option in their chart, then it's out of our control.

**Comment by iorlas** [comment]
Aah I see. It might be a bit confusing since only a subset of minio values is defined in the values.yaml and "additionalProperties: true" has slipped out of my sight. Worth adding a comment about it in the values.yaml :D

Gonna check it later tomorrow, ty!

**Comment by BHunter2889** [comment]
Just want to revive this... the Minio chart is indeed deprecated, but also it is now official archived and obsolete. 

I had thought that would've been prioritized over the Mongo update personally given than it was already marked for deprecation when we cloned this chart in early March. It works okay for now though so I can understand the delay.

**Comment by tico24** [comment]
My main problem is time. But the other issue is that the official minio operator sucks. It is over-complicated for our needs. I need to look at the bitnami operator when I have time. PRs are of course welcome though.

**Comment by tico24** [comment]
tl;dr This is not something I will be personally taking on. Pull requests are welcome, but a migration path will be needed.

- The operator helm chart is too bloaty to include within sorry-cypress, plus it'll conflict if people already have it installed.
- So we could reference an external minio, and include a slimline default minio install.
